README.md

```diff
@@ -30,7 +30,7 @@ and permissions on Google Cloud.
 
 ```yaml
 jobs:
-  run:
+  job_id:
```

Comment:
Can you tell me more about this change? Why is this necessary and what does it achieve?

Reply:
If someone was to copy-paste that code into a GitHub action, then the following error message would be thrown: "A job requires either runs-on or uses"